ocs_ci/ocs/constants.py

```diff
@@ -314,7 +314,8 @@
 )
 DEFAULT_VOLUMESNAPSHOTCLASS_CEPHFS_MS = f"{DEFAULT_CLUSTERNAME}-cephfs"
 DEFAULT_VOLUMESNAPSHOTCLASS_RBD_MS = f"{DEFAULT_CLUSTERNAME}-ceph-rbd"
-
+DEFAULT_VOLUMESNAPSHOTCLASS_RBD_PC = f"{DEFAULT_CLUSTERNAME}-ceph-rbd"
```

Comment:
These constants are the same as DEFAULT_VOLUMESNAPSHOTCLASS_CEPHFS_MS and DEFAULT_VOLUMESNAPSHOTCLASS_RBD_MS, right?

Reply:
yes

Reply:
Would it make sense to reuse them? They seem to be the same volume snapshot classes.

Reply:
may be to reuse it , we need to rename the existing one . doing it. Either a way can work. I see not used in multiple places so change is feasible.